src/compute_horde_facilitator_sdk/sdk.py

```diff
@@ -0,0 +1,103 @@
+from typing import Any
+
+import httpx
+
+
+def acquire_token(base_url: str, username: str, password: str) -> str:
+    with httpx.Client() as client:
+        response = client.post(f"{base_url}/api-token-auth/", data={"username": username, "password": password})
+    response.raise_for_status()
+    return response.json()["token"]
+
+
+async def async_acquire_token(base_url: str, username: str, password: str) -> str:
+    async with httpx.AsyncClient() as client:
+        response = await client.post(f"{base_url}/api-token-auth/", data={"username": username, "password": password})
+    response.raise_for_status()
+    return response.json()["token"]
```

Comment:
can we make these part of the client?

if tokens would be short lived, or user just wanted to keep using username:password, they will keep creating 2 HTTP connections, one for auth and one for everything else

Reply:
token does not expire - you can generate it once and use instead of user and password - if we need short lived tokens I can easily add some JWT tokens with expiration, access tokens, refresh tokens etc... I even thought about not implementing those functions so user would not try to regenerate token on every usage

Reply:
I think they should get the token from the website (after logging in)